roles/gateway_configure_processing_node/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Autoprovision primary database gateway node.
```

Comment:
need to update the name, it is for a processing node

Reply:
Corrected. Thanks.